docs/reference-guide/security.md

```diff
@@ -128,3 +128,23 @@ done by issuing the following command:
 ```cypher
   SHOW PRIVILEGES FOR user_or_role;
 ```
+
+## Owners
+
+The privileges of the owners of
+[streams](/reference-guide/streams/overview.md#creating-a-stream) and
+[triggers](/database-functionalities/triggers.md#owner) are propagated to the
+corressponding query executions:
+- in case of streams for the queries returned by the transformations
+- in case of triggers for trigger statements
+
+This means the execution of the queries will fail if the owner doesn't have the
+required privileges. There are a few important details:
+- If there are no existing users, no privilege check is performed similarly to
+regular queries.
+- If a stream or trigger is created without using a logged in user
+session, the owner will be `Null`. From the point when the first user is created
```

Comment:
`Null` or `NULL`?

Reply:
It is shown as `Null`, that why I use `Null` instead `NULL`. Though I haven't checked how it is used in other places. Will check.